vignettes/SetSpecies.Rmd

```diff
@@ -0,0 +1,182 @@
+---
+title: "Mosquito species"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Mosquito species}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(malariasimulation)
+```
+    
+### Plotting functions
+We will create a few plotting functions to visualise the output.
+```{r}
+cols <- c("#E69F00", "#56B4E9", "#009E73", 
+          "#F0E442", "#0072B2", "#D55E00", "#CC79A7")
+
+# Plotting functions
+plot_prev <- function() {
+  plot(x = output$timestep, y = output$n_detect_730_3650 / output$n_730_3650, 
+       type = "l", col = cols[3], lwd = 2.5,
+       xlab = "Time (days)", ylab = expression(paste(italic(Pf),"PR"[2-10])),
+       xaxs = "i", yaxs = "i", bty = "l", ylim = c(0,1))
+  lines(x = output_all$timestep, y = output_all$n_detect_730_3650 / output_all$n_730_3650,
+        col = cols[5], lwd = 2.5)
+  abline(v = sprayingtimesteps, lty = 2, lwd = 2.5, col = "black")
+  grid(lty = 2, col = "grey80", lwd = 1)
+  legend("bottomleft", box.lty = 0, legend = c("Spraying", "Prevalence for species-specific scenario", "Prevalence for default all mosquitoes scenario"),
+         col = c("black", cols[3], cols[5]), lty = c(2,1,1), lwd = 2.5)
+}
+```
+    
+As alluded to in the Model Structure, Vector Control: IRS, and Vector Control: Bed net vignettes, it is possible to account for varying proportions of mosquito species in the setting you are modelling using the `set_species()` function. IRS and bed nets could be expected to have different impacts depending on the proportion of each mosquito species because of variations in indoor resting, insecticide resistance, and proportion of bites on humans versus animals by species. If you have specified more than 1 species, then the arguments for `set_spraying()` and/or `set_bednets()` must be populated with values for each species at each timestep that the intervention is implemented. 
+
+There are preset parameters for *An. gambiae*, *An. arabiensis*, and *An. funestus* that can be set by the helper functions `gamb_params()`, `arab_params()`, and `fun_params()`, respectively. The default values for each species in these helper functions are from [Sherrard-Smith et al., 2018](https://doi.org/10.1038/s41467-018-07357-w). The parameters are:  
+
+* `blood_meal_rates`: the blood meal rates for each species 
+* `foraging_time`: time spent taking blood meals    
+* `Q0`: proportion of blood meals taken on humans  
+* `phi_bednets`: proportion of bites taken in bed    
+* `phi_indoors`: proportion of bites taken indoors    
+
+We will demonstrate how to specify different mosquito species and how this could alter intervention impact using an example with IRS.
+
+## IRS with default mosquito settings
```

Comment:
I'm undecided as to the value of this additional example, I'd vote for doing one of the following:

1. Just removing this example and leaving the vignette only up to line 52, or
2. Altering the example so that it compares 2 runs, each with a single species, but with one bionomics parameter varying - eg. example species 1 has low indoor resting and example species 2 has high indoor resting. then I think you could simplify the outputs to a single plot showing the differnt impact of IRS in each example.

Reply:
Have chosen option 2 here